update/genesis/trieExport/inactiveTrieExport.go

```diff
@@ -0,0 +1,70 @@
+package trieExport
+
+import (
+	"context"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	"github.com/ElrondNetwork/elrond-go/common"
+	"github.com/ElrondNetwork/elrond-go/state"
+)
+
+type inactiveTrieExport struct {
```

Comment:
this is not quite inactive as it does have implementation. We should rename it to something like "processingTrieExport"